CustomKnight/Skin/Base/Skinable/InvSD.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace CustomKnight
+{
+    public class InvSD : Skinable_Sprite
+    {
+        public static string Name = "CRHeart";
+        public InvSD() : base(Name) { }
+        public override void SaveDefaultTexture()
+        {
+            GameObject sd = GameCameras.instance.hudCamera.gameObject.FindGameObjectInChildren("Inventory").FindGameObjectInChildren("Inv").FindGameObjectInChildren("Equipment").FindGameObjectInChildren("Super Dash");
+            ckTex.defaultSprite = CreateSpritewithppu(SpriteUtils.ExtractTextureFromSpriteLegacy(sd.GetComponent<SpriteRenderer>().sprite), sd.GetComponent<SpriteRenderer>().sprite.pixelsPerUnit);
```

Comment:
we shouldn't have it be extracted every time, instead we should include these files in the default skin ( which we just re-apply when ck is turned off.)

Reply:
actually, we should just save a reference to the existing sprite and reapply it ( without creating a new sprite from it in ApplySprite that would allow us to reset sprites without needing to extract it from the atlas